evaluator/src/main/kotlin/Rule.kt

```diff
@@ -105,6 +105,11 @@ abstract class Rule(
      */
     abstract fun issueSource(): String
 
+    /**
+     * Add an issue of the given [severity] for [pkgId] to the list of violations. Optionally, the offending [license]
+     * and [source][licenseSource] of the license can be specified. The [message] further explains the violation itself
```

Comment:
Did you mean `[licenseSource]` instead of `[source][licenseSource]`?

Reply:
Actually, no. Just "source" reads nicer as part of the text, but I still want to link that text to `licenseSource`.

Reply:
ok, thanks!

Reply:
Suggestion: `...the offending [license] and its [source][licenseSource] can be specified...`

Reply:
Done.